src/js/src/components/SearchSingleItemComponents/SearchSingleItemAllData.vue

```diff
@@ -120,12 +118,8 @@ export default {
       this.allDataShown = !this.allDataShown
       let searchString = attribute + ':"' + encodeURIComponent(value) + '"'
```

Comment:
Done. However I gotta ask - does it have any implications at all to change this? I know it can't be altered when you declare it with const, but sometimes i fail to see the difference in practice.